fdbserver/Status.actor.cpp

```diff
@@ -2255,6 +2265,23 @@ ACTOR static Future<JsonBuilderObject> workloadStatusFetcher(
 
 		(*qos)["throttled_tags"] = throttledTagsObj;
 
+		if (SERVER_KNOBS->GLOBAL_TAG_THROTTLING) {
+			JsonBuilderObject globalTagThrottlerObj;
+			for (const auto& [tag, tagStats] : globalTagThrottlerStatus.status) {
+				JsonBuilderObject tagStatsObj;
+				tagStatsObj["desired_tps"] = tagStats.desiredTps;
+				tagStatsObj["reserved_tps"] = tagStats.reservedTps;
+				if (tagStats.limitingTps.present()) {
+					tagStatsObj["limiting_tps"] = tagStats.limitingTps.get();
+				} else {
+					tagStatsObj["limiting_tps"] = "<unset>"_sr;
+				}
+				tagStatsObj["per_client_tps"] = tagStats.perClientTps;
+				globalTagThrottlerObj[printable(tag)] = tagStatsObj;
+			}
```

Comment:
Not a big deal, but I think It's better to use a separate function here to wrap the code since this function is already very long... And the long function is somewhat a bad style. I want to make sure we start to improve it from now on